Готова восьмая проектная работа

EvgeniyKorobitsin opened the pull request:
Здравствуйте!
Направляю на ревью проектную работу

EvgeniyKorobitsin committed:
Add files via upload

EvgeniyKorobitsin committed:
Готовый проект

EvgeniyKorobitsin committed:
Исправил переносы на новую строку

AliaksandrHrychonak commented on index.html:
```diff
@@ -49,63 +49,455 @@ <h1 class="logo__title">закрывающий тег</h1>
+  <main class="content container">
+
+    <section class="posts flex-column">
+      <article class="card">
```
Отличное сематическое решение использовать `<article>`

AliaksandrHrychonak commented on index.html:
```diff
@@ -49,63 +49,455 @@ <h1 class="logo__title">закрывающий тег</h1>
+
+          <div class="card__buttons">
+            <button type="button" class="button card__icon-button">
+              <svg xmlns="http://www.w3.org/2000/svg" width="39" height="36" viewBox="0 0 39 36" fill="none"
```
Можно лучше

Данный элемент не несёт смысловой нагрузки, было бы хорошим решением добавить `aria-hidden="true"`.

Подробнее => https://doka.guide/a11y/aria-hidden/

AliaksandrHrychonak commented on index.html:
```diff
@@ -49,63 +49,455 @@ <h1 class="logo__title">закрывающий тег</h1>
+
+  <dialog id="dialog-id" class="dialog">
+    <form method="dialog" class="dialog__form grid">
+      <svg class="dialog__icon" aria-hidden="true">
```
Можно лучше

Иконка и параграф не относятся к форме, будет лучше вынести их из тега `<form>`

AliaksandrHrychonak commented on index.html:
```diff
@@ -49,63 +49,455 @@ <h1 class="logo__title">закрывающий тег</h1>
+      <p class="dialog__text">
+        Вставьте дискету, чтобы продолжить
+      </p>
+      <button type="button" class="button animate-button" onclick="window['dialog-id'].close()">
```
Надо исправить

Кнопка отвечает за отправку формы, используйте значение `submit`

AliaksandrHrychonak commented on styles/style.css:
```diff
@@ -158,23 +161,112 @@
+  inline-size: 21px;
+}
+
+@supports((text-stroke: 1px var(--label-contour-color)) or (-webkit-text-stroke: 1px var(--label-contour-color))) {
```
Отлично, описана проверка поддержки `text-stroke`

AliaksandrHrychonak commented on styles/style.css:
```diff
@@ -251,27 +344,87 @@
+  padding: 30px 42px;
+}
+
+.dialog::backdrop {
```
Отлично 

Фон стилизован через псевдокласс `::backdrop`

AliaksandrHrychonak reviewed (commented):
Необходимо развернуть общий комментарий ⬇️

Добрый день. Работа выполнена на хорошем уровне! 

Отлично:

* реализованы все необходимые состояния интерактивных элементов
* отлично реализован фоновый градиент
* Текстовое содержимое плавно перекрашивается
* Фильтры реализованы, есть множественный фильтр
* аккуратно оформленный код
* все комментарии "Отлично"

Надо исправить:

* Все комментарии с пометкой "надо исправить"
* Проект необходимо выложить на `gh pages` и добавить ссылку в `README`
* В проекте есть нарушения в отображении по макету
    * Правая и верхняя координата лейбла карточки на 375 https://disk.yandex.com/i/420BNmFl4uTyKg
    * Для более точного контроля отображения вёрстки можно воспользоваться плагином `PerfectPixel` для `Chrome`, можно в фигме выделить весь макет, экспортировать его картинкой и наложить на вёрстку в браузере. Плагин позволяет наложить изображение на вёрстку в браузере, тем самым можно слегкостью подправить недочёты по отображению. 


Можно лучше:

* Все комментарии "Можно лучше" в коде.
* Будет лучше что бы на адаптиве края карточки не прижимались к краям экрана, можно воспользовать функцией расчёта для ширины карточки(следите что бы текст карточек оставался в читаемом виде, например в строке не оставалось по 1-2 слова)
    
Необходимо исправить все "надо исправить". Исправление "можно лучше" остаются на ваше усмотрение.

EvgeniyKorobitsin committed:
Устранил замечания

EvgeniyKorobitsin committed:
Add files via upload

EvgeniyKorobitsin committed:
Create README.md

AliaksandrHrychonak reviewed (commented):
Добрый день, отличные исправления, осталось подправить

- На 1440px не по макету https://disk.yandex.com/i/CMIVN0yoGFaPTw

EvgeniyKorobitsin committed:
Отработал последний комментарий ревьюера

В style.css удалил
.content {
    margin-block-start: 90px;
} 
из медиа-запроса @media (width >= 1440px)
